File: C++/Owca.cpp
#include "Owca.h"

Owca::Owca(Swiat& swiat, int x, int y) :
	Zwierze(swiat, x, y)
{
	this->sila= 4;
	this->inicjatywa= 4;
	this->znak = OWCA;
}

Owca::~Owca()
{
}

File: Java/Owca.java
import java.awt.*;

public class Owca extends Zwierze{
    public Owca(Swiat swiat, int x, int y) {
        super(swiat, x, y);
        this.sila= 4;
        this.inicjatywa= 4;
        this.znak = TypOrganizmu.OWCA;
        setKolor(new Color(255, 153, 204));
    }
}
